src/facebook_example/bot.clj

```diff
@@ -28,6 +29,17 @@
       ; If no rules apply echo the user's message-text input
       :else (reaction/echo message-text))))
 
+(defn on-quick-reply [event]
+  (println "on-quickreply event:")
```

Comment:
Yeah I would leave it like this so we do not have to print variables corresponding to each event, which would become less clear for logging.